src/util/devices.ts

```diff
@@ -53,7 +53,16 @@ export const defaultCore: CoreDefinition = {
     protectionStatus: 'PROTECTION_STATUS_SECURE_REGIONS',
 };
 
-type DeviceFamily = 'nRF51' | 'nRF52' | 'nRF53' | 'nRF91' | 'Unknown';
+/**
+ * Supported Nordic device families
+ */
+export enum DeviceFamily {
```

Comment:
Why change this to an enum? Because the values are used in apis that are not scoped to DeviceFamily (e.g. usagedata)